src/editor/editor.js

```diff
@@ -156,9 +156,11 @@ export default (model, actions) =>
         const content = file.content || model.linkPatched[file.url] || ''
             , mode = modes[file.name.split('.').pop()] || 'javascript'
 
+        const editable = (model.state.editable === false || file.editable === false) ? false : true
```

Comment:
Yeah almost there :P Still should be editable if file.editable is not set, so this should do it
```
const editable = model.state.editable && file.editable !== false
```

Reply:
I think it already is, but your code is better, so I'll push that in a minute.

Reply:
I'm sorry, you're absolutely right! Fine to keep it as is ;)